### PR TITLE
[VAN-126702] SqlLike request fix to retrieve two types of the DTOs

### PR DESCRIPTION
**Pull request by Shmalikov**
Updated mustache file for generate oneOf schema. 
Added script to simplify local debugging.

### CODE REVIEW

File: sdk-blueprints/update-schema.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Internal script for debugging SDK generation locally.
+# Run from the root of the openapi-clients repository.
+
+# Define directory where projects are stored
+projects_dir="$HOME/projects"
+
+vserver_dir="$projects_dir/com.visier.vserver"
+python_sdk_repository_dir="$projects_dir/python-sdk/src"
+
+version="22222222.99201.1494"
```
Comment (hareskoug) [inline sdk-blueprints/update-schema.sh:10]: Should not have hard-coded versions in the file. Use `$1` and error if not provided

File: sdk-blueprints/update-schema.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+# Before running this script docker image should be build from repository com.visier.containers.openapi
+docker container run -u $UID --rm -v "$vserver_dir":/vserver -e VERSION=$version -e OPENAPI_OUTPUT_ROOT="/vserver/target/openapi" -e GENERATE_API_DESCRIPTIONS=true visier-openapi:latest
```
Comment (hareskoug) [inline sdk-blueprints/update-schema.sh:13]: No point in generating descriptions for this is there?
```suggestion
docker container run -u $UID --rm -v "$vserver_dir":/vserver -e VERSION=$version -e OPENAPI_OUTPUT_ROOT="/vserver/target/openapi" -e GENERATE_API_DESCRIPTIONS=false visier-openapi:latest
```

File: sdk-blueprints/update-schema.sh [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# Internal script for debugging SDK generation locally.
+# Run from the root of the openapi-clients repository.
+
+# Define directory where projects are stored
+projects_dir="$HOME/projects"
```
Comment (hareskoug) [inline sdk-blueprints/update-schema.sh:5]: Consider this for parameterization as well (optional `$2` with default as the current value)

File: sdk-blueprints/update-schema.sh (hunk 1 above)
Comment (Shmalikov) [reply sdk-blueprints/update-schema.sh]: I added version as a parameter, but set 0.0.001 version as default to make this script easy to call.

File: sdk-blueprints/update-schema.sh (hunk 2 above)
Comment (Shmalikov) [reply sdk-blueprints/update-schema.sh]: set GENERATE_API_DESCRIPTIONS=false

File: sdk-blueprints/update-schema.sh (hunk 3 above)
Comment (Shmalikov) [reply sdk-blueprints/update-schema.sh]: Added projects_dir as an argument with default value.

**Review by hareskoug — APPROVED** [review]

**Pull request merged**